runtime/lua/vim/lsp/_snippet.lua

```diff
@@ -204,6 +204,84 @@ function Node:__tostring()
   return table.concat(insert_text, '')
 end
 
+local M = {}
+
+local function new(t)
+  return setmetatable(t, Node)
+end
+
+function M.Snippet(children)
```

Comment:
please follow the conventions used by the rest of the codebase. We don't TitleCase method names.

Reply:
Ofc :+1: